client/register-businesses.py

```diff
@@ -65,7 +119,8 @@ def random_business():
     :return:
     """
     print("adding business image to remote: ")
-    image_cid = Ipfs.add(ICON_PATH, args.ipfs_local)
+    print(f"IPFS IS: {IPFS_LOCAL}")
```

Comment:
this print statement should have been removed, I just forgot it because I wanted to make sure I have to declare the global variable twice.. forgot to remove it